examples/estimators/compute_terminal_states_gpcca.md

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
Is this intentional that you've added this file? If not, please remove.

Reply:
Yes, I removed it again as it was not intentional.